pkg/apis/policy/v1beta1/clusterimagepolicy_validation.go

```diff
@@ -23,6 +23,7 @@ import (
 	"strings"
 
 	"github.com/aws/aws-sdk-go-v2/aws/arn"
+	"github.com/sigstore/cosign/cmd/cosign/cli/options"
```

Comment:
I'd really really really not like to bring in cosign cli here.
Could we use things from here?
https://pkg.go.dev/crypto#pkg-constants

Or from sigstore/sigstore if they exist there?

Reply:
Yes, good call.

Reply:
I checked sigstore and there is not a clear list of support algorithm, as those identified in the cosign repo. So I decided to use our own validation, similar to the one in cosign here.

Reply:
Crypto package didn't have a function to validate the algorithm name, as those supported in cosign.